src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -61,7 +61,7 @@ namespace Uno.UI {
 				const fromFrame = fromProgress * Lottie._numberOfFrames;
 				const toFrame = toProgress * Lottie._numberOfFrames;
 
-				a.playSegments([fromFrame, toFrame], false);
+				a.playSegments([fromFrame, toFrame], true);
```

Comment:
It's a force flag to ensure the animation doesn't wait until the current segment is done playing. It was causing some issues with delays in animations. Actually, I have to check if it will work properly without it now that some other bugs were fixed.

Reply:
Yeah we definitely need it, without the force flag the animation plays in its entirety before playing the proper segment.

Reply:
Please document this in the code.